src/server.cpp

```diff
@@ -917,9 +919,19 @@ void Server::AsyncRunStep(bool initial_step)
 						&far_players, disable_single_change_sending ? 5 : 30);
 				break;
 			case MEET_BLOCK_NODE_METADATA_CHANGED:
-				infostream << "Server: MEET_BLOCK_NODE_METADATA_CHANGED" << std::endl;
+				{
```

Comment:
seems there is already a profiler line... didn't see it, it was hidden by an identation level lol